template_feed/Microsoft.DotNet.Web.ProjectTemplates.2.0/content/StarterWeb-CSharp/Controllers/ManageController.cs

```diff
@@ -274,19 +300,23 @@ public IActionResult SetPassword()
             });
         }
 
+        //
+        // POST: /Manage/LinkLogin
         [HttpPost]
         [ValidateAntiForgeryToken]
         public async Task<IActionResult> LinkLogin(string provider)
         {
             // Clear the existing external cookie to ensure a clean login process
-            await HttpContext.SignOutAsync(IdentityCookieOptions.ExternalScheme);
+            await HttpContext.Authentication.SignOutAsync(_externalCookieScheme);
```

Comment:
Constant needs to be updated to: await HttpContext.SignOutAsync(IdentityConstants.ExternalScheme)